pkg/model/stage.go

```diff
@@ -91,6 +91,9 @@ const (
 	// StageECSCanaryClean represents the stage where
 	// the CANARY variant resources has been cleaned.
 	StageECSCanaryClean Stage = "ECS_CANARY_CLEAN"
+	// StageCustomSync represents the stage where users can
+	// run their defined scripts as part of the deployment.
```

Comment:
```suggestion
	// StageCustomSync represents the stage where users can use their
	// defined scripts to sync the application's state instead of the KIND_SYNC stage.
```